docs/study/build-tour.py

```
@@ -0,0 +1,248 @@
+        sys.exit(1)
+    print(f"   Generated {OUTPUT_TEX.name}")
+
+    # Post-process the LaTeX
+    print("5. Post-processing LaTeX...")
+    postprocess_tex(OUTPUT_TEX)
+    print("   Added red boxes, small margins, fitted graphics")
+
+    # Run xelatex
+    print("6. Running xelatex...")
+    if run_xelatex(OUTPUT_TEX):
+        print(f"\n=== Success! ===")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)